Matchmaker/README.md

```diff
@@ -1,12 +1,6 @@
-# Pixel Streaming Matchmaker
+### Where is matchmaker?
+The matchmaker has been removed. It was not a good refernce for how to handle Pixel Streaming at scale and was adding to maintenance work, so we made the decision to remove it and discontinue its support.
 
-Instead of having all users connect to the same stream, you may want each person to end up in their own interactive experiences. To do this, you can run a separate stack of Pixel Streaming components for each user, and direct each user to a separate Signaling and Web Server to start a connection.
+### What if I still need Matchmaker?
+The matchmaker can be obtained from older releases or other branches. It was removed in preparation for UE5.5 so anything below that will have it.
```

Comment:
Maybe add a link for easier navigation:

It is removed for UE5.5, but any previous version will still have it, latest one being [UE 5.4 matchmaker](
https://github.com/EpicGamesExt/PixelStreamingInfrastructure/tree/UE5.4/Matchmaker).